main.js

```diff
@@ -0,0 +1,143 @@
+        { className: 'Backspace', Backspace: 'Backspace', },
+    ],
+    // row2
+    [{ className: 'Tab', Tab: 'Tab' },
+        { className: 'KeyQ', en: ['q', 'Q'], },
+        { className: 'KeyW', en: ['w', 'W'], },
+        { className: 'KeyE', en: ['e', 'E'], },
+        { className: 'KeyR', en: ['r', 'R'], },
+        { className: 'KeyT', en: ['t', 'T'], },
+        { className: 'KeyY', en: ['y', 'Y'], },
+        { className: 'KeyU', en: ['u', 'U'], },
+        { className: 'KeyI', en: ['i', 'I'], },
+        { className: 'KeyO', en: ['o', 'O'], },
+        { className: 'KeyP', en: ['p', 'P'], },
+        { className: 'BracketLeft', en: ['[', '{'], },
+        { className: 'BracketRight', en: [']', '}'], },
+        { className: 'Backslash', en: ['\\', '|'], },
+        { className: 'Delete', Del: 'Del' }
+    ],
+    // row3
+    [{ className: 'CapsLock', CapsLock: 'Caps Lock' },
+        { className: 'KeyA', en: ['a', 'A'] },
+        { className: 'KeyS', en: ['s', 'S'] },
+        { className: 'KeyD', en: ['d', 'D'] },
+        { className: 'KeyF', en: ['f', 'F'] },
+        { className: 'KeyG', en: ['g', 'G'] },
+        { className: 'KeyH', en: ['h', 'H'] },
+        { className: 'KeyJ', en: ['j', 'J'] },
+        { className: 'KeyK', en: ['k', 'K'] },
+        { className: 'KeyL', en: ['l', 'L'] },
+        { className: 'Semicolon', en: [';', ':'] },
+        { className: 'Quote', en: ['\'', '"'] },
+        { className: 'Enter', Enter: 'Enter' }
+    ],
+    // row4
+    [{ className: 'ShiftLeft', Shift: 'Shift' },
+        { className: 'KeyZ', en: ['z', 'Z'] },
+        { className: 'KeyX', en: ['x', 'X'] },
+        { className: 'KeyC', en: ['c', 'C'] },
+        { className: 'KeyV', en: ['v', 'V'] },
+        { className: 'KeyB', en: ['b', 'B'] },
+        { className: 'KeyN', en: ['n', 'N'] },
+        { className: 'KeyM', en: ['m', 'M'] },
+        { className: 'Comma', en: [',', '<'] },
+        { className: 'Period', en: ['.', '>'] },
+        { className: 'Slash', en: ['/', '?'] },
+        { className: 'ArrowUp', up: '▲' },
+        { className: 'ShiftRight', Shift: 'Shift' }
+    ],
+    // row5
+    [{ className: 'ControlLeft', Ctrl: 'Ctrl' },
+        { className: 'Meta', Win: 'Win' },
+        { className: 'AltLeft', Alt: 'Alt' },
+        { className: 'Space', Space: 'Space' },
+        { className: 'AltRight', Alt: 'Alt' },
+        { className: 'ControlRight', Ctrl: 'Ctrl' },
+        { className: 'ArrowLeft', left: '◄' },
+        { className: 'ArrowDown', down: '▼' },
+        { className: 'ArrowRight', right: '►' }
+    ],
+];
+
+class KeyboardKey {
+    constructor(options) {
+        const keyButton = document.createElement('div');
+        keyButton.classList.add('keyboard-key');
+        keyButton.classList.add(options.className);
+        if (Object.prototype.hasOwnProperty.call(options, 'en') && options.className !== 'Language') {
+            keyButton.insertAdjacentHTML('afterbegin', options.en[0]);
```

Comment:
insted of `options.en[0]` you could use variable witj current language, see line 19.
`options[Language][0]` - it will be helpful in case more than one language on keyboard.